### PR TITLE
Add 2021q2 entry for libpmc/hwpmc work

### PR DESCRIPTION
**Pull request by mhorne**

### CODE REVIEW

File: 2021q2/pmc.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+== Performance Monitoring Counters
+
+Contact: Mitchell Horne <mhorne@FreeBSD.org>  
+
+This quarter, some improvements were made to the `pmc(3)` library and
+`hwpmc(4)` kernel module.
+
+Background: Traditionally, PMC event codes have been defined manually in the
+`sys/dev/hwpmc/pmc_events.h` header, and compiled statically into the kernel.
```
Comment (pauamma) [inline 2021q2/pmc.adoc:9]: IIRC, 2-item lists have no comma before "and".

File: 2021q2/pmc.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+These definitions provide a translation between an event's name and its
+encoding, and are leveraged by PMC tools like `pmcstat(8)` via the
```
Comment (pauamma) [inline 2021q2/pmc.adoc:11]: Ditto.

File: 2021q2/pmc.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+to arm64. Some minor refactoring to libpmc was performed, making porting to
+other platforms in the future easier. A few small bugs were found and fixed
+along the way; notably, the "instructions" alias for `pmcstat(8)` should work
+again on Intel CPUs. The ability was added to query the legacy event tables
+when pmu-events does not contain the desired event, restoring the ability to
+use the `pmc.soft(3)` events on x86. Finally, event definitions for newer AMD
+Zen3 CPUs were imported (credits: Greg V).
+
+An larger update of the PMU event definitions is planned for Q3 of 2021. This
+will bring us in sync with 3 years of updates for x86 CPUs, and greatly
```
Comment (pauamma) [inline 2021q2/pmc.adoc:28]: And again.

File: 2021q2/pmc.adoc (hunk 1 above)
Comment (mhorne) [reply 2021q2/pmc.adoc]: Thanks, seems to be a bad habit of mine :)

**Pull request closed without merging**